R/io-parsers.R

```diff
@@ -615,23 +631,29 @@ parse_mixcr <- function(.filename, .mode) {
   .freq <- "Proportion"
   df$Proportion <- df[[.count]] / sum(df[[.count]], na.rm = TRUE)
 
-  .aa.seq <- IMMCOL$cdr3aa
-  df[[.aa.seq]] <- bunch_translate(df[[.nuc.seq]])
+  .aa.seq.cdr1 <- IMMCOL_EXT$cdr1aa
```

Comment:
Maybe I missed something, but where is declaration of IMMCOL_EXT and what is it?

Reply:
It's in immunr_data_format.R:
```
IMMCOL_EXT$cdr1nt <- "CDR1.nt"
IMMCOL_EXT$cdr1aa <- "CDR1.aa"
IMMCOL_EXT$cdr2nt <- "CDR2.nt"
IMMCOL_EXT$cdr2aa <- "CDR2.aa"
```